src/edfi-performance-test/edfi_performance_test/api/client/student.py

```diff
@@ -470,7 +467,7 @@ def create_with_dependencies(self, **kwargs):
             gradingPeriodReference__periodSequence=period_reference["attributes"][
                 "periodSequence"
             ],
-            objectiveCompetencyObjectiveReference__objective=objective_reference[
+            competencyObjectiveReference__objective=objective_reference[
```

Comment:
One of many examples where something was simply wrong, perhaps because the original programmer was looking at the Data Standard instead of the API schema, and thus did not realize that the `objective` prefix would be removed.